homeassistant/components/zwave_js/sensor.py

```diff
@@ -172,11 +170,6 @@ class ZwaveSensorEntityDescription(SensorEntityDescription):
         device_class=DEVICE_CLASS_TEMPERATURE,
         state_class=STATE_CLASS_MEASUREMENT,
     ),
-    ENTITY_DESC_KEY_TIMESTAMP: ZwaveSensorEntityDescription(
-        ENTITY_DESC_KEY_TIMESTAMP,
-        device_class=DEVICE_CLASS_TIMESTAMP,
-        state_class=STATE_CLASS_MEASUREMENT,
-    ),
     ENTITY_DESC_KEY_TARGET_TEMPERATURE: ZwaveSensorEntityDescription(
         ENTITY_DESC_KEY_TARGET_TEMPERATURE,
         device_class=DEVICE_CLASS_TEMPERATURE,
```

Comment:
Do we want some basic checking of the device and state class in the tests ?

Reply:
This is being removed, as it has been detected by this PR: #52671
Once that PR is merged, it is at least being detected (doesn't fully answer/fulfill your question, yet, attributes maybe 😬 )

Reply:
It's good to add tests for all the device classes and units but I don't think it's a requirement to test all now as long as we have tests for all the logic. I think we can work on adding more tests as we get more test data.